spec/pulmap/diacritics_spec.rb

```diff
@@ -12,8 +12,11 @@ def title_query_params q
     { qf: "${title_qf}", pf: "${title_pf}", q: q }
   end
   before(:all) do
+    solr(
+      port: ENV['CI'] ? "8983" : ENV['lando_pulmap_test_solr_conn_port'],
+      core: ENV['CI'] ? "solr/pulmap-core" : "solr/blacklight-core",
+    )
```

Comment:
😱

Reply:
Yeah, I'm making another ticket for this, sorry for the tech. debt. I already started messing with it just now.

Reply:
Totally cool, it makes the code better overall, we can nitpick this line later